src/components/navbar.js

```diff
@@ -39,6 +39,9 @@ export default class NavBar extends React.Component {
     super(props)
 
     this.toggle = this.toggle.bind(this)
+    this.fixed = props.fixed
```

Comment:
Since these are props and we only need them in the render method, do they need to be set as class attributes?

Reply:
How are they to be accessed in the render method otherwise?